packages/expo/package.json

```diff
@@ -69,6 +69,7 @@
     "expo-constants-interface": "~1.1.0",
     "expo-contacts": "~1.1.0",
     "expo-core": "~1.2.0",
+    "expo-errors": "^0.0.1",
```

Comment:
Does it make sense to start at `0.0.1` and not `1.0.0` like Unimodules?

Reply:
I think so until we feel that this library is stabilized, maybe once we have finished the audit.